lib/common/strings.c

```diff
@@ -300,10 +300,10 @@ pcmk__scan_double(const char *text, double *result, const char *default_text,
  * \internal
  * \brief Parse a guint from a string stored in a hash table
  *
- * \param[in]  table        Hash table to search
- * \param[in]  key          Hash table key to use to retrieve string
- * \param[in]  default_val  What to use if key has no entry in table
- * \param[out] result       If not NULL, where to store parsed integer
+ * \param[in,out] table        Hash table to search
```

Comment:
Do we modify `table` or anything in it?

Reply:
Another one that goes back to the library. g_hash_table_lookup() doesn't use const, so this is being conservative. Arguable though